lib/pengine/rules.c

```diff
@@ -130,11 +130,6 @@ find_expression_type(xmlNode * expr)
 
     } else if (pcmk__str_eq(attr, CRM_ATTR_ROLE, pcmk__str_none)) {
         return role_expr;
-
-#if ENABLE_VERSIONED_ATTRS
-    } else if (pcmk__str_eq(attr, CRM_ATTR_RA_VERSION, pcmk__str_none)) {
-        return version_expr;
```

Comment:
We can add a deprecation note to `expression_type:version_expr`

Reply:
good catch